apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -201,3 +201,4 @@ const ExampleFormTemplate: TemplateFunction<CheckboxProps> = ({
 
 export const Default = createStory<CheckboxProps>(Template, defaultArgs)();
 export const ExampleForm = createStory<CheckboxProps>(ExampleFormTemplate, defaultArgs)();
+export const ExampleFormChecked = createStory<CheckboxProps>(ExampleFormTemplate, { ...defaultArgs, checked: true })();
```

Comment:
I don't think this story is necessary. Please can you remove it? We can just set checked to true and reload the page if we want to manually test the checked on page load